lib/mkDummySrc.nix

```
@@ -195,6 +196,7 @@ let
       '' + optionalString (trimmedCargoToml ? package) ''
         # To build regular and dev dependencies (cargo build + cargo test)
         ${cpDummy parentDir "src/lib.rs"}
+        ${cpDummy parentDir "src/bin/dummy/main.rs"}
```

Perhaps we should name the dummy binary as `crane-dummy` so it's a _bit_ more obvious to someone where it could be coming from if they are poking through their `target` directory

```suggestion
        ${cpDummy parentDir "src/bin/crane-dummy/main.rs"}
```